.github/workflows/biweekly-build-test-and-publish-release.yaml

```
@@ -0,0 +1,81 @@
+# Copyright 2024 Dynatrace LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Bi-weekly build, test and publish Release
+
+on:
+  schedule:
+    # At 01:00 UTC on Friday
+    - cron: '0 1 * * 5'
+  workflow_dispatch:
+
+env:
+#  The first run of the action should be scheduled for a date that falls on the same weekday as specified in the cron.
+#  For example, if the cron is set to run on Fridays, the first run should be on a Friday.
+#  Additionally, during every second week of the bi-weekly cycle, the action will be skipped.
+  FIRST_RUN_DATE: 2024-09-20
+
+jobs:
+  biweekly-build-test-and-publish-release:
+    runs-on: ubuntu-latest
+    outputs:
+      weekIndex: ${{ steps.biWeeklyTagCreate.outputs.weekIndex }}
+    steps:
+      - name: Checkout source code of the repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Bi-weekly new tag
```

```suggestion
      - name: Create a new tag bi-weekly
```
Please use imperative.